python/sidecar/http.py

```diff
@@ -35,16 +35,22 @@
 
 # Sidecar API
 # -----------
-@sidecar_api.post("/ingest")
+@sidecar_api.get("/")
+async def http_index():
+    return {"message": "Hello World"}
+
+
+@sidecar_api.post("/ingest/{project_id}")
 async def http_ingest(project_id: str) -> IngestResponse:
     user_id = "user1"
     project_path = projects.get_project_path(user_id, project_id)
-    req = IngestRequest(pdf_directory=project_path)
+    uploads_dir = project_path / "uploads"
+    req = IngestRequest(pdf_directory=str(uploads_dir))
     response = ingest.run_ingest(req)
     return response
 
 
-@sidecar_api.get("/ingest_status")
+@sidecar_api.get("/ingest_status/")
```

Comment:
Maybe we made this take {project_id} as well?

Reply:
Maybe not. check the other comment.